nms/app/fbcnms-projects/magmalte/app/theme/default.js

```diff
@@ -420,30 +420,13 @@ export default createMuiTheme({
         padding: '8px 16px',
         color: colors.primary.brightGray,
         backgroundColor: colors.primary.white,
-        '&$notchedOutline': {
-          borderColor: colors.primary.gullGray,
-        },
-        '&$hovered $notchedOutline': {
-          borderColor: colors.secondary.malibu,
-          borderWidth: '1px',
-        },
-        '&$focused $notchedOutline': {
-          borderColor: colors.secondary.dodgerBlue,
-          borderWidth: '2px',
-        },
-        '&$disabled': {
-          borderColor: colors.button.lightOutline,
-        },
```

Comment:
This all was related to when we split from symphony. These are some general styles pulled from their default styles and have no effect on our styles. It also seems that some of the identifiers being used `&$focused` and `&$hovered` don't actually work and as such were returning errors. I think this may have been a part of an older spec for Material UI. For targeting `hover` and `focus` state you'd now use `&:hover` and `&:focus` respectively. As such they had no bearing on the current design and could be removed.